client/knowledge_base_search/src/containers/ResultContainer/ResultContainer.jsx

```diff
@@ -1,9 +1,36 @@
 import React from 'react';
 import classNames from 'classnames';
 import { string, number, node, oneOfType } from 'prop-types';
+import { Icon } from 'watson-react-components';
 import './styles.css';
 
-function ResultContainer({ resultText, resultRank, children }) {
+function ResultContainer({ resultText, resultRank, originalRank, children }) {
+
+  function getRankDisplay(resultRank, originalRank) {
+    if (resultRank < originalRank) {
```

Comment:
we could probably just use some sort of "up", "down", and "equal" check here and use that to set the className + text instead of repeating code